main.py

```diff
@@ -0,0 +1,1605 @@
+                                 foreground='white', activeforeground='white')
+    terminal_enter_button.pack(padx=(0,5), pady=(0,8), ipadx=15, fill=None, expand=FALSE, side=LEFT)
+    button_config(terminal_enter_button)
+
+    #clear terminal button
+    terminal_clear_button = Button(terminal_input_frame, text='Clear', command=terminal_clear) 
+    terminal_clear_button.config(activebackground='#345', activeforeground='white')
+    terminal_clear_button.pack(padx=(0,5), pady=(0,8), ipadx=15, fill=None, expand=FALSE, side=LEFT)
+    button_config(terminal_clear_button)
+
+    #bind "Enter" key event with terminal_input
+    terminal_input.bind('<Return>', terminal_enter_event)  #bind enter key with event function
+
+    '''
+    For the user terminal textbox,
+    If the user types in something that is saved/valid for our 
+    program, we will print something out on the terminal.
+    (i.e. /help will output all possible commands for our interface
+          /listdevices will output all devices in that account
+          /accountinfo will output mqtt account info
+          etc.)
+    
+    If the user types in something we don't know, just do nothing.
+    Everytime the user hits "enter", print out what was typed and clear the entry box
+    '''
+
+    for widget in frame.winfo_children():
+        widget.grid(padx=5, pady=3)
+    return frame
+
+def create_left_frame(container):
+    global device_list
+    global device_info
+    global select_message
+
+    frame = Frame(container)
+    frame.columnconfigure(0, weight=1)      #one column
+    for i in range(7):      
+        frame.rowconfigure(i, weight=1)    #eight rows
+    
+    frame.option_add("*Background", 'white')
+    frame.option_add("*Foreground", 'black')
+
+    '''
+    #nRF Cloud logo *****INCOMPLETE, try to make transparent
+    nrfLogo = ImageTk.PhotoImage(Image.open("nrfcloud_logo.png"))
+    logo = Label(frame, image=nrfLogo)
+    logo.config(background='#748587')
+    logo.grid(row=0, column=0, padx=(5), pady=(5), sticky=W+E+N+S)
+    '''
+
+    #(0,0) select device drop down menu at top left
+    device_options = []
+    device_list = StringVar()
+    device_list.trace('w', change_device)
+    device_list.set("Select Device...")
+   
+    device_options.append('Select Device...')
+    for device in resp2['items']:
+        if device['id'].startswith('nrf-'): #only include devices and not account device
+            device_options.append(device['id'])    #put device IDs in device_list array
+
+    select_device_frame = Frame(frame, borderwidth=2, relief="ridge")
+    select_device_frame.grid(column=0, row=0, padx=10, pady=(10,0), sticky=W+E+N+S)
+
+    select_device = OptionMenu(select_device_frame, device_list, *device_options)  #dropdown menu
+    select_device.config(font=(myFont, 12), background=nordic_blue, foreground='white')
+    select_device['menu'].configure(bg='#d6d6d6', bd=0, font=(myFont, 12))
```

Comment:
Eventually it would be nice to see the hard-coded background colors here and below replaced with names, similar to the colors up at the top of this file.